mathesar_ui/src/component-library/button/Button.scss

```diff
@@ -11,6 +11,8 @@
   font-weight: inherit;
   position: relative;
   padding: 0.5em 0.5em;
+  gap: 0.25em;
```

Comment:
Because [browser support for Flexbox gap](https://caniuse.com/flexbox-gap) is still not great, we've decided to only allow usage of `gap` with `display: grid` in our codebase. We shouldn't use `gap` with `display: flex` or `display: inline-flex`.